.github/workflows/compile.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: Compile rules
+
+on:
+  - pull_request
+  - push
```

Comment:
In other places I've limited push to just the default branch. Would that make sense here as well?

Reply:
Probably yes (as I use the foreman repo directly for PRs, not a fork)